### PR TITLE
Readme badges

### PR DESCRIPTION
**Pull request by owensgit**
# Description

I though it would be nice to add some badges to our README, let me know what you think. Happy to add/remove or just bin them all 🙂 

## Requirements

Take a look at the README in GitHub.

Storybook link: n/a
Design link: n/a
Screenshot:
Screenshot 2021-03-12 at 16 19 21

# Checklist:

- [x] The commit messages follow the [convention for this project](./blob/master/docs/CONTRIBUTING.md#conventional-commits)
- [ ] I have provided an adequate amount of test coverage
- [ ] I have added the functionality to the [test app](./blob/master/docs/CONTRIBUTING.md#build-test-application)
- [ ] I have provided a story in storybook to document the changes
- [ ] I have provided documentation in the notes section of the story
- [ ] I have added any new public feature modules to public-api.ts
- [ ] I have [linked the new component](<(https://github.com/Legal-and-General/canopy/blob/master/docs/CONTRIBUTING.md#invision-dsm)>) to adobe DSM (if appropriate)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+[![Open Source Love svg1](https://badges.frapsoft.com/os/v1/open-source.svg?v=103)](https://github.com/ellerbrock/open-source-badges/)
+[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
+[![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://GitHub.com/Naereen/StrapDown.js/graphs/commit-activity)
```
Comment (andy-polhill) [inline README.md:4]: I think you are supposed to link to your own commit activity 😁  but I'm not sure this one brings too much value anyway.

**Comment by andy-polhill** [comment]
I quite like the idea of some badges, but have a bit of concern about where they come from and how useful some of them are. The one provided by GitHub actions for example is a live badge in that it's not just static content.

Screenshot 2021-03-12 at 16 27 20

File: README.md (hunk 1 above)
Comment (owensgit) [reply README.md:4]: Whoops 🤦  good spot

File: README.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+[![Open Source Love svg1](https://badges.frapsoft.com/os/v1/open-source.svg?v=103)](https://github.com/ellerbrock/open-source-badges/)
```
Comment (andy-polhill) [inline README.md:1]: I'm a bit hesitant about linking to images on someone else's server, It's probably ok but images can be used for analytics and other slightly nefarious activities.

**Comment by andy-polhill** [comment]
Netlify have badges too, which could be useful in the README to highlight the status of the current production deploy.
https://docs.netlify.com/monitor-sites/status-badges/

File: README.md (hunk 2 above)
Comment (owensgit) [reply README.md:1]: Yes that's a good point, probably better safe than sorry

Comment (andy-polhill) [reply README.md:1]: on reading up I think there is a notion of `marketing badges` and then badges that provide a bit more contextual information, I think those contextual ones might provide a bit more value, what do you think?

**Comment by andy-polhill** [comment]
@dependabot badge me

File: README.md (hunk 2 above)
Comment (owensgit) [reply README.md:1]: Yes I think you're right, some dynamic badges with live information about the project would be more useful. I'll add the Github actions one, and also look into the Netlify one - and then perhaps look into others like coverage, etc.

**Comment by owensgit** [comment]
Closing this PR for now as we're going to adopt form different badges instead, and won't get around to it just yet.

**Pull request closed without merging**